### ISSUE TITLE
docs: add section for pre release

### ISSUE THREAD

**Issue by chelnak** [chelnak/gh-changelog#128]

**Comment by ssbarnea** [comment]
I was surprised too to discover that unreleased changes are not grouped into sections, even if these were properly configured.

**Comment by chelnak** [comment]
Yeah this was a design decision I made at the start as I wanted to follow a certain changelog style.

I'm open to this as a feature though. It could be configurable.
